assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -106,6 +107,32 @@ const Collapse = styled.button`
   }
 `;
 
+function Toggle({ children, toggle, ...rest }) {
+  // We unfortunately have to manually assign this listener, as it would be default behaviour
+  // if it wasn't for our listener further up the stack interpreting enter as "enter edit mode"
```

Comment:
I wonder if instead, the "enter edit mode" could be adjusted so that it would be ignored when relevant input is in focus. E.g. in a similar way as `isEditableTarget` or `nativeCopyPasteExpected` are working. Thoughts?

Reply:
That would be an option too - but not easy to do. I don't think I'll look into that option in this PR, but it's definitely an option for improvement.